### PR TITLE
SAAS-3758: Added Oracle User API keys Rotated plugin, spec

### PR DESCRIPTION
**Pull request by AkhtarAmir**

### CODE REVIEW

File: plugins/oracle/identity/usersMfaEnabled.js [hunk 1]
```diff
@@ -8,7 +8,7 @@ module.exports = {
     more_info: 'User accounts should have an MFA device setup to enable two-factor authentication.',
     link: 'https://docs.cloud.oracle.com/iaas/Content/Identity/Tasks/usingmfa.htm',
     recommended_action: 'Enable an MFA device for the user account.',
-    apis: ['user:list'],
+    apis: ['user:list', 'apiKey:list'],
```
Comment (dipsubha06) [inline plugins/oracle/identity/usersMfaEnabled.js:11]: Do we need the second api call, are we using it any where

File: plugins/oracle/identity/userAPIKeysRotated.js [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+
+        if (!users || !apiKeys) return callback(null, results, source);
+
+        if (users.err || !users.data) {
+            helpers.addResult(results, 3,
+                'Unable to query for users: ' + helpers.addError(users), 'global');
+            return callback(null, results, source);
+        }
+
+        if (!users.data.length) {
```
Comment (dipsubha06) [inline plugins/oracle/identity/userAPIKeysRotated.js:44]: are we using users data?

File: plugins/oracle/identity/userAPIKeysRotated.js [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+            helpers.addResult(results, 0, 'No user accounts found', 'global');
+            return callback(null, results, source);
+        }
```
Comment (AkhtarAmir) [inline plugins/oracle/identity/userAPIKeysRotated.js:47]:
```suggestion
```

File: plugins/oracle/identity/userAPIKeysRotated.js [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+
```
Comment (AkhtarAmir) [inline plugins/oracle/identity/userAPIKeysRotated.js:44]:
```suggestion
```

File: plugins/oracle/identity/usersMfaEnabled.js (hunk 1 above)
Comment (AkhtarAmir) [reply plugins/oracle/identity/usersMfaEnabled.js:11]: Yes we are

File: plugins/oracle/identity/usersMfaEnabled.js [hunk 5]
```diff
@@ -33,6 +33,7 @@ module.exports = {
             warn_federated_users: settings.warn_federated_users || this.setting.warn_federated_users.default
         };
         var warnFedUsers = (config.warn_federated_users == 'true');
+        console.log(JSON.stringify(cache, null, 2));
```
Comment (AkhtarAmir) [inline plugins/oracle/identity/usersMfaEnabled.js:36]:
```suggestion
```

File: plugins/oracle/identity/usersMfaEnabled.js (hunk 1 above)
Comment (AkhtarAmir) [inline plugins/oracle/identity/usersMfaEnabled.js:11]:
```suggestion
    apis: ['user:list'],
```

**Pull request merged**